src/lambda_alerting.py

```diff
@@ -55,6 +55,12 @@ def lambda_handler(event, context):
     settings = Settings.from_s3_path(settings_s3_path)
 
     resource_type = ResourceTypeResolver.resolve(event)
+    if not resource_type:
+        logger.info(
+            f"Event for the source {event['source']} and detail-type {event['detail-type']} is not alertable."
```

Comment:
maybe we can rephrase from "is not alertable" to something like "are not handled by SALMON"?
To distinguish those from events which we can process, but by business logic they are not-alertable?

Reply:
sure, thank you!